code/signup.html

```diff
@@ -0,0 +1,126 @@
+
+    <div class="navigation">
+
+      <div class="TabletDesktopMenu">
+        <a class="menuLink" href="./index.html">Home</a>
+        <a class="menuLink" href="./signup.html">Sign up</a>
+      </div>
+      <!-- Mobile Hamburger menu start -->
+      <div>
+        <button class="ham"></button>
+        <nav class="navbar">
+          <ul>
+            <li><a class="menuLink" href="./index.html">Home</a></li>
+            <li><a class="menuLink" href="./signup.html">Sign up</a></li>
+          </ul>
+        </nav>
+      </div>
+      <!-- Mobile hamburger manu end -->
+
+      <!-- Header title start -->
+      <div>
+        <h2>What the stock</h2>
+      </div>
+      <!-- Header title end -->
+
+    </div>
+
+    <!-- Hero image and text start -->
+    <div class="heroContainer">
+      <!-- Hero Image -->
+      <img src="./images/heroImage.jpg" alt="" class="headerImage">
+
+      <div class="heroTitle">
+        <!-- Header Title -->
+        <h1>Use the form below to sign up</h1>
+
+      </div>
+    </div>
+    <!-- Hero image and text end -->
+  </div>
+  <!-- Header container end -->
+
+
+
+  <!-- Signup form start -->
+  <div class="signupFormFlexbox">
+    <div class="signupFormStyling">
+      <form action="https://httpbin.org/anything" type="POST">
+        <h1 style="text-align: center; padding-bottom: 10px;">Sign up</h1>
+          <center>
+            <input type="text" id="firstName" name="firstName" placeholder="First Name:" required><br>
+            <input type="text" id="lastName" name="lastName" placeholder="Last Name:" required><br>
+            <input type="email" id="email" name="email" placeholder="E-mail:" required><br>
+            <input type="password" id="password" name="password" placeholder="Password:" required><br>
+          </center>
+  
+        <h3 class="signupFormGenderTitle" style="text-align: center;">Gender:</h3>
+            <label for="">
+              <input type="radio" id="male" name="gender" value="male" required>
+              Male
+            </label>
+            
+            <label for="">
+              <input type="radio" id="female" name="gender" value="female" required>
+              Female
+            </label>
+  
+            <label for="">
+              <input type="radio" id="other" name="gender" value="other" required>
+              Other
+            </label>
+  
+        <h3 style="text-align: center;">Interests:</h3>
+          <div class="signupFormInterestsFlexbox">
+
+        <label for="">
+          <input type="checkbox" id="interest1" name="interest1" value="html">
+          HTML
+        </label>
+
+        <label for="">
+          <input type="checkbox" id="interest2" name="interest2" value="css" style="margin-left: 20px;">
+          CSS
+        </label>
+
+        <label for="">
+          <input type="checkbox" id="interest3" name="interest3" value="javascript" style="margin-left: 20px">
+          Javascript
+        </label>
```

Comment:
Nice with checkboxes! The `name` can be the same as the `value`, it will make more sense when you look at the HTTPbin summary. Also, remember indentation :)

Reply:
I'm not sure you're using the `id`s of these